tests/lib/rules/array-bracket-newline.js

```diff
@@ -40,6 +40,14 @@ ruleTester.run("array-bracket-newline", rule, {
         "var foo = [/* \nany comment\n */];",
         "var foo = [/* single line multiline comment for no real reason */];",
         "var foo = [[1,2]]",
+        {
+            code: "var foo = [[1,2]]",
+            options: [{ }]
+        },
+        {
+            code: "var foo = [[1,2]]",
+            options: []
+        },
```

Comment:
just to confirm: this is not to test the change, but to improve the test coverage?
(as the tester will not throw error, even if the `options` was modified)

Reply:
ops, i think it might dup by mistake I will remove this case. 

I have added `[{}]` cases for those which have default values in properties

Reply:
`"added [{}] cases "` to improve the test coverage?

Reply:
> `"added [{}] cases "` to improve the test coverage?

true, 

should I add some more [here](https://github.com/anikethsaha/eslint/tree/fixes-12592/tests/fixtures/config-file/cloned-config) ?

Reply:
it's great to improve the test coverage, but it seems out the scope of this PR. maybe another PR, thoughts?

Reply:
sorry for not getting, can you mention which one you are suggesting ?

I think [this one](https://github.com/anikethsaha/eslint/tree/fixes-12592/tests/fixtures/config-file/cloned-config) is in the scope of this PR right ?

Reply:
sure, I mean the new added tests in rules.

Reply:
done 👍